crates/cxx-qt-lib/src/core/qset/qset_qpersistentmodelindex.rs

```diff
@@ -0,0 +1,76 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
```

Comment:
This should maybe also be updated to 2023 :thinking: 
That does mean we have to adjust the generate.sh though, which will change it in all files, even the ones we didn't touch. :/

Reply:
These are effectively "binary" / machine generated files rather than source files, so the `generate.sh` is the one that needs the correct copyright. And worst case the date is a start date so it's probably not worth the effort :thinking:

Reply:
Yeah, would assume an earlier date here is better if in doubt :)